client/ayon_core/host/dirmap.py

```diff
@@ -118,7 +118,7 @@ def get_mappings(self):
             site, in that case configuration in Local Settings takes precedence
         """
 
-        dirmap_label = "{}-dirmap".format(self.host_name)
+        dirmap_label = "{}_dirmap".format(self.host_name)
```

Comment:
Shouldn't that be just `"dirmap"`?

Reply:
See the comment in the PR description - it could be, but if we do that then we'd also need to update `ayon-maya` to match that.

The question is - does Nuke dirmapping currently work? Because it seems it doesn't actually use the `core` dirmapping functionality but rolls its own. Which would mean that if we would make this get mappings just use `dirmap` it may suddenly incorrectly assume it should do that for Nuke too. I don't know about the Nuke dirmapping myself though, maybe @jakubjezek001 @moonyuet know?

Reply:
Now it seems to be broken at multiple places, I would rather use `dirmap` which is pre-determined key without host name. And for cases when host name cannot be used as settings model attribute name (e.g. if starting with number)...

Reply:
> And for cases when host name cannot be used as settings model attribute name (e.g. if starting with number)...

Wouldn't that case still break the parent key we're using in project settings? :P Anyway, happy to refactor - will make a matching PR, someone would then need to confirm the Nuke behavior however.

Reply:
> The question is - does Nuke dirmapping currently work? Because it seems it doesn't actually use the `core` dirmapping functionality but rolls its own. Which would mean that if we would make this get mappings just use `dirmap` it may suddenly incorrectly assume it should do that for Nuke too. I don't know about the Nuke dirmapping myself though, maybe @jakubjezek001 @moonyuet know?
Works with nuke-based dirmap setting and it is added to the callback. 
![image](https://github.com/user-attachments/assets/c1fc7b25-2c07-457e-9819-e41c74b70ff5)
But it doesn't work with the core-based dirmap setting so guess the core one only hooked to the maya host or?

Reply:
@jakubjezek001 may be able to elaborate on the Nuke part? But it sounds like it may then remain unrelated to this PR.